libs/vkd3d/meta.c

```diff
@@ -1413,6 +1413,46 @@ void vkd3d_execute_indirect_ops_cleanup(struct vkd3d_execute_indirect_ops *meta_
     pthread_mutex_destroy(&meta_indirect_ops->mutex);
 }
 
+static HRESULT vkd3d_dstorage_ops_init(struct vkd3d_dstorage_ops *dstorage_ops, struct d3d12_device *device)
+{
+    VkPushConstantRange push_range;
```

Comment:
The shaders require Int64. We should only create these meta shaders if memoryDecompression is supported.